smhi/metobs.py

```diff
@@ -102,7 +102,9 @@ def get_data(self, period: str = "corrected-archive"):
         """
         self.data = MetObsDataV1(self.period.period, period)
         self.table_raw = self.data.get()
-        self.table = pd.read_csv(io.StringIO(self.table_raw), on_bad_lines="skip")
+        data_starting_point=self.table_raw.find('Datum')
+        self.header=self.table_raw[0:data_starting_point]
+        self.table = pd.read_csv(io.StringIO(self.table_raw[data_starting_point:-1]),sep=';', on_bad_lines="skip",usecols=[0,1,2])
```

Comment:
Looks good. How is no data handled? I get back a table with a few rows of NaNs now.

Also, try using black (do `pip install black` and activate `Save on format` in VSCode). It'll format your code for you every time you save, i.e, space between `=` etc. I'll add a format check later as a GitHub Action. You could also try pre-commit (also `pip install pre-commit`), which does some checks every time you commit. We have a `.pre-commit-config.yaml` file in the repo root which should be used if you do `pre-commit install`. This won't let you commit until some checks are fulfilled :).

Reply:
Also, I see the intergation test fails. Try to update the test when you make changes to the code. To run a test locally it should be enough to write `pytest` if it is installed (also with pip).

Reply:
Thanks - will update tomorrow. Have installed the suggested packages, will look into VSCode settings tomorrow with the work.